backend/dataline/main.py

```diff
@@ -122,8 +122,9 @@ async def execute_sql(
 
 @app.get("/toggle-save-query/{result_id}")
 async def toggle_save_query(result_id: str) -> SuccessResponse[None]:
-    db.toggle_save_query(result_id=result_id)
-    return SuccessResponse()
+    with db.DatabaseManager() as conn:
+        db.toggle_save_query(result_id=result_id, conn=conn)
+        return SuccessResponse()
```

Comment:
Lets change this to use repos while we're at it. Time to take out allthis shit code

Reply:
Upon further investigation, I noticed that "toggle_save_query" is not used anywhere on the frontend, and the table `SavedQueryModel` is obsolete. Should I remove it and anything related to it?

Reply:
Oh.. right I removed this a while back in favor of just storing edits.

We're going to add it later when we do dashboards and saved queries and stuff. But yeah just remove it, we will find it in the history when it's time to reimplement